docs/source/examples/vision/guides/formatters_validating.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>"Check requires" - maybe we should say "deepchecks requires" or such</p><p>Also think we can drop the "results" from the two titles.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>